lib/lang/en.js

```diff
@@ -163,6 +163,9 @@ module.exports = {
 
     return str;
   },
-  "unavailable-radar": "Next hour forecasts are temporarily unavailable due to $1.",
+  "unavailable-radar": "Next hour forecasts are $1 unavailable due to $2.",
+  "temporarily": "temporarily",
+  "partially": "partially",
   "station-offline": "all nearby radar stations being offline",
+  "station-spotty": "nearby radar stations reporting incomplete data sets",
```

Comment:
I would prefer `"station-incomplete": "gaps in coverage from nearby radar stations"`

Reply:
(Please sanity check with @w1bw @novall @cailees )

Reply:
I also prefer "gaps in coverage from nearby radar stations".